### DISCUSSION TITLE
How to use the listings-items-api to update item quantity

### DISCUSSION THREAD

**Discussion by yupingwyp** [amzn/selling-partner-api-models#3163] [General]
I read that: https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/listings-items-api-use-case-guide/listings-items-api-use-case-guide_2020-09-01.md#step-1-submit-listings-item-patch-request

says:
"Partially Update Listings: The Listings Items API accepts PATCH operations to update one or more individual attributes for an existing listing, such as for updating price and quantity. Additionally, the Listings Items API accepts PATCH operations to delete one or more individual attributes for an existing listing."

Can you please provide a code example of how to use path operations to update the quantity? Thanks so much!

**Comment by luckdragon** [comment]
did you ever figure this out? I'm trying to figure out the same thing.

**Comment by sungolivia** [comment]
I think you are looking at the old documentation!
Did you check out this section? https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-use-case-guide#step-1-submit-listings-item-patch-request

![Screen Shot 2022-09-22 at 6 46 07 PM](https://user-images.githubusercontent.com/7606170/191878563-77f6c5a0-b726-48ab-a972-bcc70ac9662d.png)
